mtp_api/settings/base.py

```diff
@@ -1,32 +1,26 @@
 """
 Django settings for mtp_api project.
 
-For more information on this file, see
-https://docs.djangoproject.com/en/1.7/topics/settings/
-
 For the full list of settings and their values, see
-https://docs.djangoproject.com/en/1.7/ref/settings/
+https://docs.djangoproject.com/en/1.8/ref/settings/
 """
 import os
 import sys
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(__file__))
-
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
In the other apps, we're doing `from os.path import abspath, dirname, join` and then `dirname(dirname())`. Worth switching this for consistency? (I'm not fussed either way…)